rocketpy/sensors/sensors.py

```diff
@@ -8,6 +9,265 @@
 class Sensors(ABC):
     """Abstract class for sensors
 
+    Attributes
+    ----------
+    type : str
+        Type of the sensor (e.g. Accelerometer, Gyroscope).
```

Comment:
1. "type" is a python reserved variable name, we should not use it for variables.
2. If you realy needs to use it, the convention states that you use `type_` instead (that's weird, right?). But I'm sure you cannot
3. This is more critical: you never defined the type variable in the Sensors class. Apperantly you have used in the `Barometer` child class, so you can document it there. Either you initialize the type attribute with a default value in this class or you simply delete it from the documentation.

Reply:
This attribute is not used, it was just a merging error